backend/pkg/auth/auth.go

```diff
@@ -0,0 +1,117 @@
+package auth
+
+import (
+	// "fmt"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
```

Comment:
This jwt implementation is no longer maintained
use this instead [github.com/golang-jwt/jwt](https://github.com/golang-jwt/jwt) - publicly maintained version of the same project